src/Framework/Sources/JavaScript/Content/chrome/storage.js

```diff
@@ -0,0 +1,39 @@
+
+'use strict';
+
+const background = require('../background-bridge');
+
+var storage = {
```

Comment:
this doesn't include change notifications - it's OK to go in steps, just making sure you are aware of this
also, the storage API should work the same way on both content and background (and even popup) - another thing that can be added later

Reply:
added `onChanged`